src/fileWriter.ts

```diff
@@ -151,21 +151,54 @@ const getHwSigningFileType = (label: PathLabel, pathType: PathTypes): string =>
     case PathTypes.PATH_WALLET_SPENDING_KEY_BYRON:
       return `${label}HWSigningFileByron_ed25519_bip32`
 
-    case PathTypes.PATH_GOVERNANCE_VOTING_ACCOUNT:
-    case PathTypes.PATH_GOVERNANCE_VOTING_KEY:
+    case PathTypes.PATH_CVOTE_ACCOUNT:
+    case PathTypes.PATH_CVOTE_KEY:
       return `${label}HWSigningFile_ed25519`
 
     default:
       return `${label}HWSigningFileShelley_ed25519`
   }
 }
 
+const hwsfileDescription = (path: BIP32Path): string => {
+  switch (classifyPath(path)) {
+    case PathTypes.PATH_WALLET_SPENDING_KEY_BYRON:
+    case PathTypes.PATH_WALLET_SPENDING_KEY_SHELLEY:
+      return 'Payment Hardware Signing File'
+
+    case PathTypes.PATH_WALLET_STAKING_KEY:
+      return 'Stake Hardware Signing File'
+
+    case PathTypes.PATH_WALLET_SPENDING_KEY_MULTISIG:
+      return 'Script Payment Hardware Signing File'
+
+    case PathTypes.PATH_WALLET_STAKING_KEY_MULTISIG:
+      return 'Script Stake Hardware Signing File'
+
+    case PathTypes.PATH_WALLET_MINTING_KEY:
+      return 'Mint Hardware Signing File'
+
+    case PathTypes.PATH_POOL_COLD_KEY:
+      return 'Stake Pool Hardware Signing File'
+
+    case PathTypes.PATH_CVOTE_KEY:
+      return 'CIP36 Vote Hardware Signing File'
```

Comment:
CIP-36 suggests `Hardware Catalyst Vote Signing File`, so I just wanted to check with you that this is intentional.

Reply:
I'd like to keep the naming consistent with other signing files, so it ends with `Hardware Signing File`, and it is more general than Catalyst (I've agreed with gitmachtl that it is perfectly OK for him to change descriptions of the generated keys after they are written to a file --- only the user knows whether the key is to be used for Catalyst or not).